### ISSUE TITLE
Can't reliably get currently used language

### ISSUE THREAD

**Issue by Nehon** [angular-translate/angular-translate#1249]
Hello,

I have an application that is using 2 locales : en and fr. I want that my fallback language is "en", and by this, I mean that if the locale sent by the browser is not found, I want to use the "en" language.

Here is my configuration : 
```
        $translateProvider.translations("en", userTranslateEn);        
        $translateProvider.translations("fr", userTranslateFr);

        $translateProvider.fallbackLanguage("en");
        $translateProvider.registerAvailableLanguageKeys(["en, fr"], {
            "en_*": "en",
            "fr_*": "fr",
            "*": "en"
        });
        $translateProvider.preferredLanguage($localeProvider.$get().id.toLowerCase());
```
In my case $localeProvider.$get().id.toLowerCase() returns "en_us".
This works as expected and the "en" language is used.

I have a directive that displays a list of available languages, and I want to highlight the currently used language.
However, when I want to find what language is actually used, doing this : 
```
var currentLanguage = $translate.use();
```
currentLanguage is set to "en_us" whereas the *real* used language is "en"
I tried with proposedLanguage() but it returns undefined.

Is there a way to reliably get the language that is actually used?
If not, IMO that's an issue in the library

**Comment by Nehon** [comment]
Found out that there is a determinePreferredLanguage() method that find what language to use depending on the browser setting (strangely enough it can return different things than localeProvider, but it's consistent with the browser settings).

But it has the same issue, use() returns en_US

**Comment by knalli** [comment]
I've made a demo: I cannot see your issue? http://plnkr.co/edit/2qbj8nkW93ludul0zc5n?p=preview

**Comment by tspaeth** [comment]
Could it be that this is fixed since 2.8.x?

**Comment by knalli** [comment]
I think so, too.
